### ISSUE TITLE
Docs do not seem to state how to for example convert a .py file to exe

### ISSUE THREAD

**Issue by Luckz** [indygreg/PyOxidizer#49]
In the docs I read through Getting Started, Managing Projects, and Packaging, and still have no idea how I would turn my somethingsimple.py into an executable beyond perhaps by copying the entire code into

> [[embedded_python_run]]
> code = ".."

**Comment by dskeithbuck** [comment]
I'm right there with you. I'd love an example of how to convert a Python Hello World script into an executable.

```python
def main():
    print("Hello World!")

if __name__ == "__main__":
    main()
```

**Comment by indygreg** [comment]
I agree that it would be useful to have explicit documentation on how to convert a Python script to an application. We may want to add an `[embedded_python_run]` mode that specializes in doing this...

**Comment by mikeatm** [comment]
@indygreg  great, i would wager that there is alot more code out there that would benefit from PyOxidizer is not yet neatly packaged on PyPi or in a pip installable module, just a file/bunch of files. Im guilty of this.

**Comment by gnprice** [comment]
In case it helps someone coming across this: there's an example in https://github.com/indygreg/PyOxidizer/issues/27#issuecomment-506298186 by @McSpidey , and another in my https://github.com/indygreg/PyOxidizer/issues/78#issuecomment-511657412 .

(I agree with everyone that it'd be useful to put an example like this in the docs.)

**Comment by feluxe** [comment]
I'm currently using `nuitka --standalone --onefile  my_script.py` to produce a single file binary `my_script`.

I came across PyOxidizer looking for a better solution, because the binary produced by nuitka has quite a bad startiup time.

PyOxidizer seem promising, but I really have a hard time trying to create a binary from `my_script.py` with it. I've been fiddling with it for an hour, searching the docs,  without luck.

For my use case something like this would be amazing: `pyoxidizer build --venv /my/venv/dir --entry my_script.py` leaving me with a `my_script`.binary.
